Include alloca.h to avoid build errors. This can be submitted to upstream.

--- rsync-3.4.0/popt/findme.c.orig	2025-01-15 11:51:37.212734325 +0100
+++ rsync-3.4.0/popt/findme.c	2025-01-15 11:51:56.641049950 +0100
@@ -6,6 +6,7 @@
    file accompanying popt source distributions, available from 
    ftp://ftp.rpm.org/pub/rpm/dist. */
 
+#include <alloca.h>
 #include "system.h"
 #include "findme.h"
 
